docs/passwords-in-config.md

```diff
@@ -46,3 +46,33 @@ docker run --env MY_PASSWORD_OS_VARIABLE=mysecretPassword promregator/promregato
 
 Further information on this can also be found in the official [Docker reference documentation](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file).
 
+## Using Encrypted Password in Configuration
+
+As an alternative for using environment variables for the password you can use encrypted password (values starting with `{cipher}`) in the configuration. 
+This allows you to check the configuration into a version control system. 
+See also the documentation about [Encryption & Decryption](http://cloud.spring.io/spring-cloud-config/spring-cloud-config.html#_encryption_and_decryption) of the Spring Cloud Config Server.
+
+To encrypt a value you can use the [Spring Cloud CLI](https://cloud.spring.io/spring-cloud-cli/) extension and encrypt the password on the command line.
+```bash
+spring encrypt mysecret --key mySecretKey
+d93dd725143e187804e4105a681706c1185c58f92086bf55aa830e61c0527060
+```
+
+Then you need to set the encryption key as environment variable so that promregator can process it:
+```bash
+export ENCRYPT_KEY=mySecretKey
```

Comment:
Do you know how to work with docker?
If yes, I think we should mention at least in a one-liner that users making use of our docker image may set this environment variable by using the `--env` parameter (see also one section above). Moreover, we also should consider to enable making use of [Docker Secrets](https://docs.docker.com/engine/swarm/secrets/) here as well (we can do so here, as the name of the environment variable is static and known at build time of the image).

If no, please just ignore my comment (and I'll create a separate issue for that).

Reply:
I added the docker insturctions. Unfortunately, I have no experience with Docker Secrets, so I can't help you with this part.